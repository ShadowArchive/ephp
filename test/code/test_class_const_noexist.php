<?php

class A {}

$a = new A;
print $a::NO_EXISTS_THIS_CONST . "\n";

