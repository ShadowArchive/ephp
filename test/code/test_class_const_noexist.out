
Fatal error: Undefined class constant 'NO_EXISTS_THIS_CONST' in {{CWD}}/test/code/test_class_const_noexist.php on line 3
